packages/eslint-config-mocha/index.js

```diff
@@ -0,0 +1,9 @@
+
+module.exports = {
+    plugins: [
+        'mocha'
+    ],
+    rules: {
+        'mocha/no-exclusive-tests': 'error'
```

Comment:
Instead of specifying this rule, you might want to extend the `recommended` set. You'll notice the only rule enabled in the recommended set is the `no-exclusive-tests` (https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/index.js#L28) but the idea is we'll adopt best mocha practices moving forward.